src/sites/cclw/components/Articles.tsx

```diff
@@ -14,6 +14,13 @@ type TArticle = {
 };
 
 const ARTICLES: TArticle[] = [
+  {
+    url: "https://docs.google.com/forms/d/e/1FAIpQLSfpo03ytni3SmEqXfFRW38qz1OGyNgN5HhQF-6fW-QcSLZ67A/viewform",
```

Comment:
I'd say we need a single reference to this form in the code - currently we have the URL it two places